app/api/attendees.py

```diff
@@ -31,11 +35,26 @@ def before_post(self, args, kwargs, data):
         """
         require_relationship(['ticket', 'event'], data)
 
-        ticket = db.session.query(Ticket).filter_by(id=int(data['ticket'])).first()
+        ticket = db.session.query(Ticket).filter_by(
+            id=int(data['ticket'])
+            ).first()
         if ticket is None:
-            raise UnprocessableEntity({'pointer': '/data/relationships/ticket'}, "Invalid Ticket")
+            raise UnprocessableEntity(
+                {'pointer': '/data/relationships/ticket'}, "Invalid Ticket"
+                )
         if ticket.event_id != int(data['event']):
-            raise UnprocessableEntity({'pointer': '/data/relationships/ticket'}, "Ticket belongs to a different Event")
+            raise UnprocessableEntity(
+                {'pointer': '/data/relationships/ticket'},
+                "Ticket belongs to a different Event"
+            )
+
+        if db.session.query(TicketHolder.id).filter_by(
+                ticket_id=int(data['ticket']), event_id=int(data['event'])
```

Comment:
I noticed it just now. IMO it would raise NoResultFound exception and return HTTP 500  when ticket doesn't exist. Please test it :)